src/client.ts

```diff
@@ -801,6 +805,68 @@ export default class ConcordiumNodeClient {
         return response;
     }
 
+    async invokeContract(
+        blockHash: string,
+        contractContext: ContractContext
+    ): Promise<InvokeContractResult | undefined> {
+        if (!isValidHash(blockHash)) {
+            throw new Error('The input was not a valid hash: ' + blockHash);
+        }
+
+        let invoker;
+        if (!contractContext.invoker) {
+            invoker = null;
+        } else if ((contractContext.invoker as Address).address) {
+            invoker = {
+                type: 'AddressAccount',
+                address: (contractContext.invoker as Address).address,
+            };
+        } else {
+            const invokerContract = contractContext.invoker as ContractAddress;
+            invoker = {
+                type: 'AddressContract',
+                address: {
+                    subindex: invokerContract.subindex.toString(),
+                    index: invokerContract.index.toString(),
+                },
+            };
+        }
+
+        const requestObject = new InvokeContractRequest();
+        requestObject.setBlockHash(blockHash);
+        requestObject.setContext(
+            stringToInt(
+                JSON.stringify({
+                    invoker,
+                    contract: {
+                        subindex: contractContext.contract.subindex.toString(),
+                        index: contractContext.contract.index.toString(),
+                    },
+                    amount: contractContext.amount.microGtuAmount.toString(),
+                    method: contractContext.method,
+                    parameter: contractContext.parameter
```

Comment:
If no `parameter` is available I think you should be able to not include this, i.e. do not set an empty string here.